service/src/main/kotlin/fi/espoo/evaka/document/archival/SärmäClientInterface.kt

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2024 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.document.archival
+
+import fi.espoo.evaka.s3.Document
+import okhttp3.ResponseBody
+
+interface SärmäClientInterface {
```

Comment:
should this be called ArchiveClientInterface or at least XArchiveClientInterface since the _interface_ is probably not espoo/särmä specific

Reply:
Tämän ja muidenkin asioiden nimeämislogiikka täytyy vielä katselmoida kertaalleen. Että missä tehdään raja yleisen "siirrä arkistoon" toiminnan ja Särmä / X-Archive spesifisen toiminnallisuuden kanssa.

Reply:
Varmaan voidaan hioa myöhemmässä vaiheessa